package.json

```diff
@@ -25,13 +25,13 @@
     "node": ">=14"
   },
   "dependencies": {
-    "@sphereon/did-uni-client": "^0.4.0",
+    "@sphereon/did-uni-client": "file:../did-uni-client/",
```

Comment:
Don't put local references in files. Ask for releases of the package and/or up the version to a to be released version.

Reply:
Okay updated it myself. Also had to update imports to use ssi-types